integrations/extensions/starter-kits/language-model-conversational-search/elasticsearch-watsonx-actions.json

```diff
@@ -105,6 +105,12 @@
                     "expression": "[\"title\", \"text\"]"
                   },
                   "skill_variable": "query_source"
+                },
+                {
+                  "value": {
+                    "expression": "${use_knn_search} ? {\n    \"field\": \"text_embedding.predicted_value\",\n    \"query_vector_builder\": {\n      \"text_embedding\": {\n        \"model_id\": $embedding_model,\n        \"model_text\": ${query_text}\n      }\n    },\n    \"k\": 10,\n    \"num_candidates\": 100\n  }: \"\""
+                  },
+                  "skill_variable": "knn_body"
```

Comment:
if a user is using `use_knn_search` , then `query_body` needs to also be "unset" or nullified, right? If not then we would end up passing both into the ES API which would not work?

Reply:
Hi Sneha, I haven't seen any issues with passing `query_body` while passing `knn_body`. Elasticsearch seems to ignore it and handle it correctly. Did you run into any issues?

Reply:
This was fixed, thanks Sneha.